docs/en/changes/changes.md

```diff
@@ -5,7 +5,9 @@
 
 #### OAP Server
 
-
+* Fix the wrong watch semantics in Kubernetes watchers, which causes heavy traffic to API server in some Kubernetes cluster,
+  we should use `Get State and Start at Most Recent` semantic instead of `Start at Exact`
+  because we don't need the changing history events, see https://kubernetes.io/docs/reference/using-api/api-concepts/#semantics-for-watch.
```

Comment:
```suggestion
  because we don't need the changing history events, see https://kubernetes.io/docs/reference/using-api/api-concepts/#semantics-for-watch.

```